book/src/connection.md

```diff
@@ -73,7 +73,7 @@ let (client_conn, server_conn) = futures_util::try_join!(
 [`futures::stream::Stream`]: https://docs.rs/futures/latest/futures/stream/trait.Stream.html
 [`futures::sink::Sink`]: https://docs.rs/futures/latest/futures/sink/trait.Sink.html
 [`MessageStream`]: https://docs.rs/zbus/3/zbus/struct.MessageStream.html
-[`ConnectionBuilder::address`]: https://docs.rs/zbus/3/zbus/struct.ConnectionBuilder.html#method.address 
+[`Builder::address`]: https://docs.rs/zbus/3/zbus/struct.ConnectionBuilder.html#method.address 
```

Comment:
The links will also need to be updated (you need to predict it based on a local build). Was this done for the other module changes? :thinking:

Reply:
I am bad at predicting it though.. is there any way to update it or check it semi-automatically? yes, we will probably need to update all the links for v4 anyway..

Reply:
`cargo doc --open` and then find the link?

Reply:
done